binaries/scheduler/main.go

```diff
@@ -7,35 +7,30 @@ import (
 	"github.com/apache/thrift/lib/go/thrift"
 	"github.com/scootdev/scoot/sched/config"
 	"github.com/scootdev/scoot/scootapi/server"
-	"golang.org/x/net/context"
 )
 
 var addr = flag.String("addr", "localhost:9090", "Bind address for api server.")
-var cfgFile = flag.String("cfg", "", "Path to sched overlay config file.")
+var cfgText = flag.String("sched_config", "", "Scheduler Configuration.")
```

Comment:
Why change this to text? Seems unwieldy. Maybe have it be textConfig or fileConfig based on contents.